pkgs/applications/office/clickup/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib, stdenv, fetchurl, appimageTools, makeWrapper, undmg, unzip }:
+
+let
+  pname = "clickup";
+  name = "ClickUp";
```

Comment:
```suggestion
```
The variable name is rather confusing because pname+version is name but I don't think we need it at all. Just hardcode the string.